### PR TITLE
Delete tox.ini

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

I think we are not using tox anymore. Earlier today I tried to run the tests with pytest, and I got an error because I have this file. Should we erase it?

FYI @laserprec 

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,104 +0,0 @@
...
-extras = all
-
-[testenv:flake8]
-deps = flake8
-skip_install = True
-commands = flake8 .
-
-
-# Configurations for running pytest
-[pytest]
```
Comment (laserprec) [inline tox.ini:56]: Tox also stores configuration related to other python tools like `pytest` and `flake8`. Maybe the error is relating them. Do you think we would need any of these configurations?

File: tox.ini [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-    gpu: mark a test as gpu test
-    spark: mark a test as spark test
-    vw: mark a test as vowpal wabbit test
-testpaths =
-    tests
-addopts =
-    # reports all (except passed tests). See https://docs.pytest.org/en/latest/usage.html#detailed-summary-report
-    -ra
-    --durations 10
-    --cov-append --cov=recommenders --cov-report=term --cov-report=xml --junitxml=junit/test-results.xml
```
Comment (laserprec) [inline tox.ini:80]: I think by default we report code coverage at the end of the tests. Maybe the error you are seeing is relating to missing the `coverage` package?

File: tox.ini (hunk 1 above)
Comment (miguelgfierro) [reply tox.ini:56]: I would ask back a question, is there anything in the tox that we can reused in the workflow that @pradnyeshjoshi did https://github.com/microsoft/recommenders/tree/main/.github/workflows

File: tox.ini (hunk 2 above)
Comment (miguelgfierro) [reply tox.ini:80]: @pradnyeshjoshi in the AzureML tests, do we have coverage report?

**Review by laserprec — APPROVED** [review]

**Pull request merged**